subiquity/client/controllers/serial.py

```diff
@@ -29,6 +29,13 @@ async def make_ui(self):
         ssh_info = await self.app.client.meta.ssh_info.GET()
         return SerialView(self, ssh_info)
 
+    def run_answers(self):
+        if 'rich' in self.answers:
+            if self.answers['rich']:
+                self.ui.body.rich_btn.base_widget._emit('click')
```

Comment:
is there a downside to, instead of emit('click'), calling down to rich_mode/basic_mode?

Reply:
Well my vague goal for "answers" is that the answers should operate the GUI as much as possible (not that we're very consistent about this). Possibly this is going a bit far...